test/controllers/user.js

```diff
@@ -127,7 +129,9 @@ describe('/POST users/sessions', () => {
           .then((res) => {
             res.should.have.status(200);
             expect(res.body).to.have.property('token');
+            token = res.body.token;
```

Comment:
Each it test should be independant from the others, so you can run them in any order or in any sub set and make them work. So you shouldn't use this token to the list endpoints